src/Shared/Grpc/ProtoUtils.cs

```diff
@@ -605,9 +608,12 @@ internal static OrchestrationStatus ToCore(this P.OrchestrationStatus status)
     /// Converts a <see cref="EntityBatchResult" /> to <see cref="P.EntityBatchResult" />.
     /// </summary>
     /// <param name="entityBatchResult">The operation result to convert.</param>
+    /// <param name="completionToken">The completion token from the work item.</param>
     /// <returns>The converted operation result.</returns>
     [return: NotNullIfNotNull(nameof(entityBatchResult))]
-    internal static P.EntityBatchResult? ToEntityBatchResult(this EntityBatchResult? entityBatchResult)
+    internal static P.EntityBatchResult? ToEntityBatchResult(
+        this EntityBatchResult? entityBatchResult,
+        string completionToken)
```

Comment:
Where is `completionToken` used in `ToEntityBatchResult()`?

Reply:
It's not being used. I accidentally left this in when working on another PR to add Durable Entity support to DTS. I can remove this.

Reply:
I've removed this. We can add it back when we finish adding entity support for DTS.